compiler/syntax/src/res_comments_table.ml

```diff
@@ -1508,7 +1508,19 @@ and walk_expression expr t comments =
         attach t.leading return_expr.pexp_loc leading;
         walk_expression return_expr t inside;
         attach t.trailing return_expr.pexp_loc trailing)
-  | _ -> ()
+  | Pexp_jsx_fragment (opening_greater_than, exprs, closing_lesser_than) ->
```

Comment:
@shulhi comments are now printed too many times.
I'm attaching them multiple times in here.
I'm not sure why.

I basically want to attach them to either `<>`, children or `</>`